### PR TITLE
control-service: add pod disruption budget

### PR DESCRIPTION
**Pull request by dakodakov**
# Why
All for higher Control Service availability while permitting the cluster administrator to manage the clusters nodes.

# What
Add optional configuration for pod disruption budget.

### CODE REVIEW

File: projects/control-service/projects/helm_charts/pipelines-control-service/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Copyright 2021-2023 VMware, Inc.
+  SPDX-License-Identifier: Apache-2.0
+ */}}
+
+{{- if .Values.poddisruptionbudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  labels: { { - include "pipelines-control-service.labels" . | nindent 4 } }
+    { { - range $key, $value := .Values.alertmanager.metadata.labels } }
```
Comment (mivanov1988) [inline projects/control-service/projects/helm_charts/pipelines-control-service/templates/poddisruptionbudget.yaml:11]: alertmanager?

Comment (dakodakov) [reply projects/control-service/projects/helm_charts/pipelines-control-service/templates/poddisruptionbudget.yaml]: fixed.

File: projects/control-service/projects/helm_charts/pipelines-control-service/templates/poddisruptionbudget.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+{{- if .Values.poddisruptionbudget.enabled }}
+apiVersion: policy/v1
+kind: PodDisruptionBudget
+metadata:
+  labels: { { - include "pipelines-control-service.labels" . | nindent 4 } }
+    { { - range $key, $value := .Values.poddisruptionbudget.metadata.labels } }
+    { { $key } }: { { $value | quote } }
+  { { - end } }
+  name: {{ .Values.poddisruptionbudget.metadata.name }}
```
Comment (mivanov1988) [inline projects/control-service/projects/helm_charts/pipelines-control-service/templates/poddisruptionbudget.yaml:14]: how about `name: {{ .Release.Name }}-pdb`

**Review by mivanov1988 — APPROVED** [review]
LGTM!

**Review by doks5 — APPROVED** [review]

**Pull request merged**